pkg/costmodel/router.go

```diff
@@ -358,6 +358,9 @@ func (a *Accesses) RefreshPricingData(w http.ResponseWriter, r *http.Request, ps
 	w.Header().Set("Access-Control-Allow-Origin", "*")
 
 	err := a.CloudProvider.DownloadPricingData()
+	if err != nil {
+		klog.V(1).Infof("Error refreshing pricing data: %s", err.Error())
```

Comment:
Should this be a `Warningf` or `Errorf` instead of `Infof`?

Reply:
Maybe? In this case I was just following the convention that was already in place e.g. [here](https://github.com/kubecost/cost-model/blob/c900910117b48d5f6886125f3d63f4f348066718/pkg/costmodel/router.go#L1572-L1575).

The error should go to the frontend anyway, this is just as a "hey look this is happening" in the kubernetes log in case you click off that settings page.